### ISSUE TITLE
#Accuracy on my dataset is 82% 

### ISSUE THREAD

**Issue by pp3993** [ZhaoJ9014/face.evoLVe.PyTorch#72]
Hi
I evaluated the performances of pre-trained model on my private Indian data-sets, where accuracy is 82% only. 
what I need to do to improve performance on my datasets.
Thank-you

**Comment by ZhaoJ9014** [comment]
> Hi
> I evaluated the performances of pre-trained model on my private Indian data-sets, where accuracy is 82% only.
> what I need to do to improve performance on my datasets.
> Thank-you

Hi, thank you for your question. Since our model is pre-train on MS-Celeb-1M, whose race source is global based, it may perform not that well on your own dataset containing Indian faces mostly (significantly different data domain). To improve the accuracy on your data, we recommend you fine-tune our released model on your own training data and perform the evaluation again. Hope this helps:)

**Comment by pp3993** [comment]
Do i need to train freshly this model on indian dataset?

**Comment by pp3993** [comment]
here, fine-tune means creation of million of indian dataset or change of hyperparameter on pretrained model?
